test/blackbox-tests/test-cases/private-modules/run.t

```diff
@@ -0,0 +1,45 @@
+  $ dune build --root accessible-via-public
+  Entering directory 'accessible-via-public'
+        runfoo alias default
+  private module bar
+
+  $ dune build --root inaccessible-in-deps 2>&1 | grep -v "cd _build"
+  Entering directory 'inaccessible-in-deps'
+        ocamlc .foo.eobjs/foo.{cmi,cmo,cmt} (exit 2)
+  File "foo.ml", line 1, characters 0-5:
+  Error: Unbound module X
+
+  $ dune build --root excluded-from-install-file
```

Comment:
I suggest to add `| grep priv`, so that the focus is on private elements

Reply:
Good idea.